notification/app/notification/services/Configuration.scala

```diff
@@ -35,4 +30,13 @@ class Configuration extends NotificationConfiguration("notification") {
 
   lazy val disableElectionNotificationsAndroid = conf.getStringProperty("notifications.elections.android.disabled", "false").toBoolean
   lazy val disableElectionNotificationsIOS = conf.getStringProperty("notifications.elections.ios.disabled", "false").toBoolean
+
+  private def getConfigurableHubConnection(maybeEndpoint: Option[String]): NotificationHubConnection = {
```

Comment:
I think this function probably should receive a string as a parameter, either "enterprise" or "newstand".

That string would then be prepended to every key when getting every property.

That would make it easier to understand
```
getConfigurableHubConnection("enterprise")
getConfigurableHubConnection("newsstand")
```